sdks/python/apache_beam/runners/dataflow/internal/apiclient.py

```diff
@@ -744,6 +744,12 @@ def translate_distribution(distribution_update, metric_update_proto):
   dist_update_proto.max = to_split_int(distribution_update.max)
   dist_update_proto.count = to_split_int(distribution_update.count)
   dist_update_proto.sum = to_split_int(distribution_update.sum)
+  if (hasattr(distribution_update, 'buckets') and
```

Comment:
<!--new_thread; commit:c89749a0d77350dfbd2686db975aa087bafb8f1b; resolved:0-->
Why wouldn't the update object have these attributes?

Also, please avoid `hasattr`: https://hynek.me/articles/hasattr/.

Reply:
We are using another distribution metric called DistributionUpdate(https://github.com/apache/beam/blob/ce0c92e0a15b9b44e53161827afe213eb18e18b4/sdks/python/apache_beam/metrics/cells.py#L367) for other metrics, which doesn't have buckets and first_bucket_offset